### ISSUE TITLE
Obtain the result index of model test.

### ISSUE THREAD

**Issue by 2110310010** [open-mmlab/mmaction2#2301]
When I use the mmaction1 v1.0,  I don't find the top_k_accuracy index in test.py.
In  https://github.com/open-mmlab/mmaction2/blob/1.x/docs/en/user_guides/4_train_test.md, I also don't find the suitable way to obtain the information. I want  to like the version of mmaction2 v0.x  to obtain these information. such as 

 Evaluating mean_class_accuracy ...

top1_acc        0.6700
top3_acc        0.9403
mean_acc        0.6292

**Comment by Dai-Wenxun** [comment]
Which config do you use?

**Comment by 2110310010** [comment]
I use the config of uniformer  model.
_base_ = ['../../_base_/default_runtime.py']

# model settings
model = dict(
    type='Recognizer3D',
    backbone=dict(
        type='UniFormer',
        depth=[5, 8, 20, 7],
        embed_dim=[64, 128, 320, 512],
        head_dim=64,
        drop_path_rate=0.3),
    cls_head=dict(
        type='I3DHead',
        dropout_ratio=0.,
        num_classes=6,
        in_channels=512,
        average_clips='prob'),
    data_preprocessor=dict(
        type='ActionDataPreprocessor',
        mean=[114.75, 114.75, 114.75],
        std=[57.375, 57.375, 57.375],
        format_shape='NCTHW'))

# dataset settings
# dataset_type = 'RawframeDataset'
# data_root = 'data/rawframes'
# data_root_val = 'data/rawframes'
# ann_file_train = 'data/train.txt'
# ann_file_val = 'data//val.txt'
# ann_file_test = 'data/val.txt'
dataset_type = 'RawframeDataset'
data_root = 'data/rawframes'
data_root_val = 'data/rawframes'
ann_file_train = 'data/train.txt'
ann_file_val = 'data/val.txt'
ann_file_test = 'data/val.txt'

# file_client_args = dict(io_backend='disk')
non_local=((0, 0, 0), (1, 0, 1, 0), (1, 0, 1, 0, 1, 0), (0, 0, 0)),
non_local_cfg=dict(
    sub_sample=True,
    use_scale=False,
    norm_cfg=dict(type='BN3d', requires_grad=True),
    mode='embedded_gaussian',
shift_div=8),

train_pipeline = [
    dict(type='SampleFrames', clip_len=8, frame_interval=2, num_clips=1),
    dict(type='RawFrameDecode'),
    dict(type='Resize',scale=(-1,256)),
    dict(type='MultiScaleCrop', 
         input_size=224,  # 网络输入
         scales=(1, 0.875, 0.75, 0.66),  
         random_crop=False, 
         max_wh_scale_gap=1), 
    dict(  # Resize 类的配置
        type='Resize',  
        scale=(224, 224), 
        keep_ratio=False), 
    dict(type='Flip', flip_ratio=0.5),
    dict(type='FormatShape', input_format='NCTHW'),
    dict(type='PackActionInputs')
]
val_pipeline = [
    # dict(type='DecordInit', **file_client_args),
    dict(
        type='SampleFrames',
        clip_len=8,
        frame_interval=2,
        num_clips=1,
        test_mode=True),
    # dict(type='DecordDecode'),
    dict(  # RawFrameDecode 类的配置
        type='RawFrameDecode'),  # 给定帧序列，加载对应帧，解码对应帧
    dict(  # Resize 类的配置
        type='Resize',  # 调整图片尺寸
        scale=(-1, 256)),  # 调整比例
    dict(type='Resize', scale=(-1, 256)),
    dict(type='CenterCrop', crop_size=224),
    dict(type='FormatShape', input_format='NCTHW'),
    dict(type='PackActionInputs')
]
test_pipeline = [
    # dict(type='DecordInit', **file_client_args),
    dict(
        type='SampleFrames',
        clip_len=8,
        frame_interval=2 ,
        num_clips=1,
        test_mode=True),
    # dict(type='DecordDecode'),
    dict(  # RawFrameDecode 类的配置
        type='RawFrameDecode'),  # 给定帧序列，加载对应帧，解码对应帧
    dict(type='Resize', scale=(-1, 256)),
    dict(type='ThreeCrop', crop_size=224),
    dict(type='FormatShape', input_format='NCTHW'),
    dict(type='PackActionInputs')
]
train_dataloader = dict(
    batch_size=2,
    num_workers=1,
    persistent_workers=True,
    sampler=dict(type='DefaultSampler', shuffle=True),
    dataset=dict(
        type=dataset_type,
        ann_file=ann_file_train,
        data_prefix=dict(img=data_root),
        # filename_tmpl='{img_:05}.jpg',
        pipeline=train_pipeline))
val_dataloader = dict(
    batch_size=2,
    num_workers=1,
    persistent_workers=True,
    sampler=dict(type='DefaultSampler', shuffle=False),
    dataset=dict(
        type=dataset_type,
        ann_file=ann_file_val,
        data_prefix=dict(img=data_root_val),
        # filename_tmpl='{img_:05}.jpg',
        pipeline=val_pipeline,
        test_mode=True))
test_dataloader = dict(
    batch_size=2,
    num_workers=1,
    persistent_workers=True,
    sampler=dict(type='DefaultSampler', shuffle=False),
    dataset=dict(
        type=dataset_type,
        ann_file=ann_file_test,
        data_prefix=dict(img=data_root_val),
        # filename_tmpl='{img_:05}.jpg',
        pipeline=test_pipeline,
        test_mode=True))

val_evaluator = dict(type='AccMetric')
# val_evaluator = dict(type='AccMetric', acc/topk=(1, 3))  # 使用分类正确率评测指标
test_evaluator = dict(type='AccMetric')

train_cfg = dict(
    type='EpochBasedTrainLoop', max_epochs=150, val_begin=1, val_interval=1)
val_cfg = dict(type='ValLoop')
test_cfg = dict(type='TestLoop')

optim_wrapper = dict(
    optimizer=dict(
        type='SGD', lr=0.005, momentum=0.9, weight_decay=1e-4, nesterov=True),
    paramwise_cfg=dict(
        custom_keys={
            '.backbone.cls_token': dict(decay_mult=0.0),
            '.backbone.pos_embed': dict(decay_mult=0.0),
            '.backbone.time_embed': dict(decay_mult=0.0)
        }),
    clip_grad=dict(max_norm=40, norm_type=2))

param_scheduler = [
    dict(
        type='MultiStepLR',
        begin=0,
        end=15,
        by_epoch=True,
        milestones=[5, 10],
        gamma=0.1)
]

default_hooks = dict(checkpoint=dict(interval=2))

# Default setting for scaling LR automatically
#   - `enable` means enable scaling LR automatically
#       or not by default.
#   - `base_batch_size` = (8 GPUs) x (8 samples per GPU).
auto_scale_lr = dict(enable=False, base_batch_size=64)
work_dir='/tf_logs/uniformer1'

**Comment by Dai-Wenxun** [comment]
val_evaluator = dict(type='AccMetric', metric_options = dict(top_k_accuracy=dict(topk=(1, 3))))

**Comment by 2110310010** [comment]
Thanks for your help! But I want to use test.py to obtain the resluts.  For example,
top1_acc 0.6700
top3_acc 0.9403
mean_acc 0.6292
In test.py file, I don't find --eval top_k_accuracy mean_class_accuracy index like v0.x version.

**Comment by Dai-Wenxun** [comment]
> val_evaluator = dict(type='AccMetric', metric_options = dict(top_k_accuracy=dict(topk=(1, 3))))

Just set the `val_evaluator` in your config
